packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/fork.controller.js

```diff
@@ -1,22 +1,435 @@
+      this.model.usePrivateNetwork = true;
+      this.model.privateNetwork = this.availablePrivateNetworks.find(
+        (network) =>
+          network.regions?.find(
+            (networkRegion) =>
+              networkRegion.openstackId === this.database.networkId,
+          ),
+      );
+      if (this.model.privateNetwork) {
+        this.initialLoadSubnet = true;
+        this.DatabaseService.getSubnets(
+          this.projectId,
+          this.model.privateNetwork.id,
+        )
+          .then((subnets) => {
+            this.availableSubnets = this.handleSubnetResponse(subnets);
+            this.model.subnet = this.availableSubnets.find(
+              (subnet) => subnet.id === this.database.subnetId,
+            );
+          })
+          .finally(() => {
+            this.initialLoadSubnet = false;
+          });
+      }
+    }
+
+    // handle query params
+    if (!this.isPITRActivated || this.backupId) {
+      this.model.restoreMode = RESTORE_MODES.BACKUP;
+    }
+
+    this.updateFlavor(this.originalFlavor);
+
+    const minDateRetentionDays = moment().subtract(
+      this.originalFlavor.availability[0].backupRetentionDays,
+      'days',
+    );
+    const clusterCreationDate = moment(this.database.createdAt);
+    const minCalendarOptionDate =
+      minDateRetentionDays > clusterCreationDate
+        ? minDateRetentionDays
+        : clusterCreationDate;
+
+    this.calendarOptions = {
+      enableSeconds: true,
+      minuteIncrement: 1,
+      maxDate: moment().toISOString(),
+      minDate: minCalendarOptionDate.toISOString(),
+    };
+
+    // If we find the parentElement, we can enable smooth scrolling. Otherwise, fallback to $anchor
+    const scrollParentElement = this.$document[0].getElementsByClassName(
+      'pci-project-content',
+    )[0];
+    this.scrollToOptions = scrollParentElement
+      ? {
+          element: scrollParentElement,
+          offset: 0,
+          horizontal: false,
+        }
+      : null;
+  }
+
+  loadMessages() {
+    this.CucCloudMessage.unSubscribe(this.messageContainer);
+    this.messageHandler = this.CucCloudMessage.subscribe(
+      this.messageContainer,
+      { onMessage: () => this.refreshMessages() },
+    );
+  }
+
+  refreshMessages() {
+    this.messages = this.messageHandler.getMessages();
+  }
+
+  get isPITRActivated() {
+    return (
+      isFeatureActivated('forkPIT', this.database.engine) ||
+      (this.database.engine === DATABASE_TYPES.MONGO_DB &&
+        this.model.plan.name === 'enterprise')
+    );
+  }
+
+  scrollTo(id) {
+    // If we have the scrollToOptions, we can use the smooth scrolling, otherwise we use $anchor
+    if (this.scrollToOptions) {
+      animateScrollTo(document.getElementById(id), this.scrollToOptions);
+    } else {
+      this.$anchorScroll(id);
+    }
+  }
+
+  onPlanChanged(plan) {
+    this.updatePlan(plan);
+  }
+
+  onFlavorChanged(flavor) {
+    this.updateFlavor(flavor);
+  }
```

Comment:
Same as https://github.com/ovh/manager/pull/8671/files#r1112860008

Reply:
Same here, we anticipated the tracking that will be added later